lib/dataContract/DataContract.js

```diff
@@ -166,7 +189,8 @@ class DataContract {
   toJSON() {
     const json = {
       $schema: this.getJsonMetaSchema(),
-      contractId: this.getId(),
+      $ownerId: this.getOwnerId(),
+      $entropy: this.entropy,
```

Comment:
Let’s add id here and move entropy out to ST. Please do not forget to check that ID is ownerId + entropy

Reply:
Decided  to  postpone it until we know how to avoid grinding attack with primary keys